### PR TITLE
Organize multimedia paths: validate

### PR DESCRIPTION
**Pull request by orangejenny**
Spec: https://docs.google.com/document/d/1fVzCgEw3-deesJqlHtBqy-CrvNvHhylmOAwODOQSdvc/edit

Download was done in https://github.com/dimagi/commcare-hq/pull/22693, this PR adds validation, and then updating the paths will be another PR.

@emord / @mkangia 

screen shot 2018-12-14 at 10 11 14 am

### CODE REVIEW

File: corehq/apps/hqmedia/static/hqmedia/js/manage_paths_main.js
Comment (emord) [inline corehq/apps/hqmedia/static/hqmedia/js/manage_paths_main.js]: Should this block merging?

**Review by emord — APPROVED** [review]
Looks like some todo's there, wasn't sure if they should be in this PR or the next one where you're updating. Approved if you're expecting to follow up

**Comment by orangejenny** [comment]
@emord The todos are for the next PR - I'd like to get this out so QA can potentially work on this piece. It's all behind a flag.

File: corehq/apps/hqmedia/static/hqmedia/js/manage_paths_main.js
Comment (orangejenny) [reply corehq/apps/hqmedia/static/hqmedia/js/manage_paths_main.js]: nah, will do in next PR

**Pull request merged**
